pkg/cosign/cosign.go

```diff
@@ -360,6 +360,7 @@ func decodeStatement(sig oci.Signature) (map[string]interface{}, string, error)
 		if err != nil {
 			return nil, "", fmt.Errorf("failed to decode statement %s: %w", string(pld), err)
 		}
+		decodedStatement["type"] = decodedStatement["predicateType"]
```

Comment:
I believe sig.Payload's return value has `predicateType` set, so I just copied that value to `type` here. How do I test that this works, are there any images to test cosign?